src/index.js

```diff
@@ -1,8 +1,33 @@
-import React from 'react';
+import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import registerServiceWorker from './registerServiceWorker';
+import { BrowserRouter, Route } from 'react-router-dom'
 
-ReactDOM.render(<App />, document.getElementById('root'));
-registerServiceWorker();
+// import App from './App';
+// import registerServiceWorker from './registerServiceWorker';
+
+// COMPONENTS
+
+import Settings from './components/settings';
+
+class App extends Component {
+
+    render(){
+        return(
+            <div> Login Page </div>
+        )
+    }
+
+}
+
+ReactDOM.render(
+    
+    <BrowserRouter>
+        <div>
```

Comment:
Is this div needed? I don't see a style applied on it so I'm not sure if this is necessary.